platforms.c

```diff
@@ -82,8 +86,9 @@ static struct pv_logger_config plat_logger_config_messages = {
 struct pv_cont_ctrl {
 	char *type;
 	void *(*start)(struct pv_platform *p, const char *rev, char *conf_file,
-		       void *data);
+		       int pipe, void *data);
```

Comment:
i dont like the var name "pipe" ... what is this? Also: what is actually in data? Is that something that could hold such logfd?

Reply:
Ok, got it, we need a better name `logfd` could be better. 
The `data` parameter is an "out" parameter, returns with the container PID